eng/mgmt/mgmtmetadata/azurearcdata_resource-manager.txt

```diff
@@ -0,0 +1,14 @@
+﻿Installing AutoRest version: v2
+AutoRest installed successfully.
+Commencing code generation
+Generating CSharp code
+Executing AutoRest command
+cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/resources/resource-manager/readme.md --csharp --version=v2 --reflect-api-versions --tag=package-features-2021-07 --csharp.output-folder=C:\WorkingSpace\AzureUx\azure-sdk-for-net\sdk\resources\Microsoft.Azure.Management.Resource\src\Generated\Features
+2021-07-19 22:48:42 UTC
+Azure-rest-api-specs repository information
+GitHub fork: Azure
+Branch:      master
+Commit:      e7682aa897902920f3a95b2f358b6f7729d18666
+AutoRest information
+Requested version: v2
+Bootstrapper version:    autorest@2.0.4413
```

Comment:
We might not need this file I believe? These APIs you are changing should belong to `resources` instead of `azurearcdata`, am I right?
And this should be an auto-generated file, we need to remove it and run the auto-generate script again, if this does not show up again, meaning that we do not need this file.

Reply:
You are right. Removed it. Can you please take another look? If it looks good to you, can you help merge it as I don't have the permission?